lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -25,7 +25,9 @@
 <script src="./dbw_tester.js"></script>
 
 <template id="links-blocking-first-paint-tmpl">
-  <link rel="stylesheet" href="./dbw_tester.css">
+  <!-- we use a relative url to test our critical-request-chains.js formatter,
+       so we are sure new URL(resource) can handle it -->
```

Comment:
for a second I thought this meant we were changing this to make the test work, not to test that the change worked :)

what about changing the comment to something like "we use a relative protocol url to test that \`new URL(resource)\` in the critical-request-chains.js formatter can handle it"?